cf-java-logging-support-core/src/test/java/com/sap/hcp/cf/logging/common/converter/DefaultPropertiesConverterTest.java

```diff
@@ -0,0 +1,205 @@
+package com.sap.hcp.cf.logging.common.converter;
+
+import static com.sap.hcp.cf.logging.common.converter.UnmarshallUtilities.unmarshal;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.Matchers.hasKey;
+import static org.hamcrest.Matchers.not;
+import static org.junit.Assert.assertThat;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.hamcrest.Matcher;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.MDC;
+
+import com.sap.hcp.cf.logging.common.Defaults;
+import com.sap.hcp.cf.logging.common.Fields;
+
+public class DefaultPropertiesConverterTest {
+
+	private static final String HACK_ATTEMPT = "}{:\",\"";
+
+	private DefaultPropertiesConverter converter;
+
+	@Before
+	public void initConverter() {
+		this.converter = new DefaultPropertiesConverter();
+	}
+
+	@Before
+	public void cleadMdc() {
+		MDC.clear();
+	}
+
+	@Test
+	public void emptyProperties() throws Exception {
+		StringBuilder sb = new StringBuilder();
+
+		converter.convert(sb, Collections.emptyMap());
+
+		assertThat(unmarshal(sb), hasDefaultProperties());
+	}
+
+	@Test
+	public void singleMdcEntry() throws Exception {
+		StringBuilder sb = new StringBuilder();
+		MDC.put("some key", "some value");
+
+		converter.convert(sb, Collections.emptyMap());
+
```

Comment:
It still should include `hasDefaultProperties`, shouldn't it?

Reply:
that is covered by the test case above.